### PR TITLE
fix config creation

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Following the instructions wouldn't work, this fixes both the instructions and configuration files. Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ public/rendered
 invited-users
 secrets
 archiver
+.datcache/
```
Comment (karissa) [inline .gitignore]: shouldn't need this anymore because archiver replaced .datcache

File: config/index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ var datadir = process.env.DATADIR || (
     ? path.join(__dirname, '..') : os.homedir())
 
 var config = {
+  archiver: '/tmp',
```
Comment (karissa) [inline config/index.js]: This is in shared.archiver, I don't think that anything at the top level is used (and if it is, it shouldn't be)

**Comment by Kriesse** [comment]
I'm still seeing .datcache files locally, would merging this help?

**Comment by karissa** [comment]
are you sure you're on master with the latest config?

**Comment by Kriesse** [comment]
@karissa ah yeah, i had the issue on `add-login-register-header`, not `master`.

**Comment by karissa** [comment]
closing this

**Pull request closed without merging**